### DISCUSSION TITLE
Error when connecting to server

### DISCUSSION THREAD

**Discussion by aleoneshahvar** [Project-MONAI/MONAILabel#456] [Q&A] [answered]
Hello! I am trying to load the deepedit model on 3D slicer and, following a video from your youtube page, downloaded the monai app on my terminal and validated its installation. However, when I refresh the server tab on slicer I get this error: Failed to fetch models from remote server. Make sure server address is correct and <server_uri>/info/ is accessible in browser. Any suggestions on what I may be doing wrong?

**Comment by SachidanandAlle** [comment, marked as answer]
you might be missing http prefix when u r using the uri in slicer..
